### PR TITLE
Pr/enable multiprotocol for cert

### PR DESCRIPTION
**Pull request by edmont**
This commit enables Bluetooth for the Thread Certification variant of the CLI sample.

It's intended to verify that it's possible to pass Thread Certification while an NCS device is advertising or connected to another BLE device.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

### CODE REVIEW

File: samples/openthread/cli/src/ble.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	BT_DATA(BT_DATA_NAME_COMPLETE, DEVICE_NAME, DEVICE_NAME_LEN),
+};
+
+static void connected(struct bt_conn *conn, uint8_t err)
+{
+	if (err) {
+		printk("Connection failed (err 0x%02x)\n", err);
```
Comment (MarekPorwisz) [inline samples/openthread/cli/src/ble.c]: won't printk cause same issues as log does? If not maybe it would be better to redirect this to shell output?

Comment (edmont) [reply samples/openthread/cli/src/ble.c]: Yes, this is only for debugging purposes. I plan to remove those logs and just "do nothing" with the connection.

File: samples/openthread/cli/harness/overlay-cert.conf [hunk 2]
```diff
@@ -16,5 +15,17 @@ CONFIG_SHELL_DEFAULT_TERMINAL_WIDTH=416
...
+# Use the precompiled OpenThread libraries
 CONFIG_OPENTHREAD_LIBRARY_1_1=y
+
+# Enable Bluetooth LE controller which supports multiprotocol
+CONFIG_BT_LL_SOFTDEVICE_DEFAULT=y
+
+# Enable and configure Bluetooth LE
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="NUS_CoAP_client"
```
Comment (MarekPorwisz) [inline samples/openthread/cli/harness/overlay-cert.conf]: CLI?

**Comment by edmont** [comment]
https://github.com/nrfconnect/sdk-nrf/pull/3191

**Pull request closed without merging**